bastionzero/target/dbtarget/dbtarget.go

```diff
@@ -54,18 +55,21 @@ func setDbTargetAttributes(ctx context.Context, schema *dbTargetModel, dbTarget
 
 	schema.IsSplitCert = types.BoolValue(dbTarget.IsSplitCert)
 	schema.DatabaseType = types.StringPointerValue(dbTarget.DatabaseType)
+	schema.DatabaseAuthenticationConfig = target.FlattenDatabaseAuthenticationConfig(ctx, dbTarget.DatabaseAuthenticationConfig)
 }
 
 func makeDbTargetDataSourceSchema(opts *target.BaseTargetDataSourceAttributeOptions) map[string]schema.Attribute {
 	dbTargetAttributes := target.BaseTargetDataSourceAttributes(targettype.Db, opts)
 	maps.Copy(dbTargetAttributes, target.BaseVirtualTargetDataSourceAttributes(targettype.Db))
 	dbTargetAttributes["is_split_cert"] = schema.BoolAttribute{
-		Computed:    true,
-		Description: "If `true`, this Db target has the split cert feature enabled; `false` otherwise.",
+		Computed:           true,
```

Comment:
Also need to add `dbTargetAttributes["database_authentication_config"] = target.DatabaseAuthenticationConfigAttribute()`